packages/tx/src/eip2930Transaction.ts

```diff
@@ -40,6 +40,11 @@ export default class AccessListEIP2930Transaction extends BaseTransaction<Access
   get transactionType(): number {
     return 1
   }
+  
+  // Alias for transactionType
+  get type() {
+    return this.transactionType
+  }
```

Comment:
If we add this here, I guess we also want to add this alias to the legacy tx where we [added type 0](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/tx/src/legacyTransaction.ts#L19) (or better: move both getters up to `BaseTransaction` and set the tx type dynamically (e.g. on a private member `_type` which is then accessed from the getters) when calling the super class. Then we are sure that these aliases are available in all subsequent tx type implementations).

Reply:
(I think it is super-crucial that we unify as much of the API as possible, so that we are not getting lost on the common API set (which I have the impression we already do a bit to some extend))

Reply:
I am e.g. still struggling with these `v`, `r`, `s` aliases `senderS()`, `senderR()` and `yParity()` which is another form of API diffusion. Still wonder if we should directly deprecate this again in favor of a unified `v`, `r` and `s` access which works across all tx formats.

Reply:
good idea I will move the type alias to the BaseTransaction so it will always be available.

yeah, the `senderS` `senderR` and `yParity` are from the [EIP-2930 spec](https://eips.ethereum.org/EIPS/eip-2930), I'm not sure why the naming was changed from the legacy spec. We could move those aliases to the BaseTransaction as well.

Reply:
Just a small note: These are called `r`, `s`, and `v` in the RPC

Reply:
i added this in https://github.com/ethereumjs/ethereumjs-monorepo/pull/1185/commits/2ea2ebc4095530e5e63a5cc983ec752c9d663a71, please review for me

Reply:
Ok, if these name remain r, s, v even in the RPC I will deprecate the `senderR`, `senderS` and `yParity` getters. This was introduced to correctly match the spec names, but I guess we loose more here than we gain by bloating the API. Will push here before review.

@ryanio changes from your additions look good, thanks! 😄